build/crds/router/router_v1_rule.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
A question, do we create those CRD files by hand or by some tools?

Reply:
The rule and ruleendpoint crd files are created automatically in hack/local-up-kubeedge.sh. If users build kubeedge by self, thet need create crd-definition files  by hand.